docs/user/runbook/index.adoc

```diff
@@ -0,0 +1,79 @@
+:page-layout: default
+:page-title: Runbook
+:page-parent: Home
+
+:experimental:
+:toc:
+
+[#get-log]
+== Get Extension Log on VS Code/VSCodium
+
+Extension log can be helpful for troubleshooting if the extension is not behaving as expected.
+To get the extension log:
+
+. Open the Command Palette (default keybind is kbd:[Ctrl+Shift+P] on Linux/Windows,
+  kbd:[Cmd+Shift+P] on macOS).
+. Type or paste in `workbench.action.showOutputChannels` and press kbd:[Enter].
+. Type or paste in `Spyglass Language Server` and press kbd:[Enter].
```

Comment:
The `spyglassmc.showOutput` command doesn't seem to work if the extension is loading or has crashed. The steps here should hopefully be more foolproof.